GETTING_STARTED.md

```diff
@@ -0,0 +1,385 @@
+
+Additionally to multi-language, `languageType` can be set to `country`. If a column has this `languageType` there needs to be another field called `countryCodes`. Here is an example of a multi-country column definition:
+
+```
+{
+  "id": 2,
+  "name": "price",
+  "kind": "currency",
+  "languageType": "country",
+  "countryCodes": [
+    "DE",
+    "AT",
+    "US"
+  ]
+  // ...
+}
+```
+
+Each value for such a multi-country currency column is a JSON object like this:
+
+```
+{
+  "DE": 47.11,
+  "AT": 47.11,
+  "US": 40.99
+}
+```
+
+#### `link`
+
+This is probably the most important column kind. It is used to make a relation between two tables. A link is a uni- or bidirectional association. Here is an example of a link column definition:
+
+```
+{
+  "id": 8,
+  "name": "country",
+  "identifier": false,
+  "kind": "link",
+  "toTable": 1, // foreign table ID
+  "toColumn": { // foreign column definition
+    "id": 1,
+    "name": "name",
+    "identifier": true,
+    "kind": "shorttext",
+    "languageType": "language",
+    "multilanguage": true
+    // ...
+  }
+  // ...
+}
+```
+
+A link column always points to a specific table — in this case table `1`. A link value is the association between at least two rows. In the following example there is one association, one link to the foreign row with the ID `13`. The value used here is the `identifier` value of the foreign row as example in the [data retrieval chapter](#21-data-retrieval). To retrieve the full row you can call `GET /tables/1/rows/13`.
+
+```
+[
+  {
+    "id": 13,
+    "value": {
+      "cs": "Česká republika",
+      "de": "Tschechien",
+      "en": "Czech Republic",
+      "en-US": "Czech Republic",
+      "es": "Chequia",
+      "fr": "République Tchèque",
+      "it": "Repubblica Ceca",
+      "pl": "Czechy"
+    }
+  }
+]
+```
+
+#### `concat` and `group`
+
+This column kinds combines multiple columns into one column. Setting multiple `identifier` columns in a table will create a `concat` column. A `concat` column combines the values from the `identifier` columns into one column and is used to reference a row in a link. The `group` column lets you combine multiple columns into one, for example grouping three columns `height`, `length`, and `depth` together into a single field for the UI as `<height> x <length> x <depth>`.
+
+#### `attachment`
+
+This column kind is used for linking files with their `uuid` to a specific cell. For more information please look up the Swagger API documentation. Can be found at `/docs` relative to your GRUD API endpoint.
```

Comment:
What about adding/retrieving files ("media" functionality in the frontend)?

Reply:
Right, `link` and `attachment` should be additional chapters. Media management is not yet covered in this document.